operator/v1/types_network.go

```diff
@@ -96,12 +96,32 @@ type DefaultNetworkDefinition struct {
 	OVNKubernetesConfig *OVNKubernetesConfig `json:"ovnKubernetesConfig,omitempty"`
 }
 
-// AdditionalNetworkDefinition configures an extra network that is available but not
+// MacVlanConfig contains configurations for macvlan interface.
+type MacVlanConfig struct {
+
+	// Host interface to enslave interface.
+	Master string `json:"master"`
+
+	// Ipam indicates which IPAM module will be used for IP Address Management(IPAM)
+	// Ipam for now supports 'dhcp'
+	// +optional
+	IPAM string `json:"ipam,omitempty"`
```

Comment:
"`ipam`" not "`Ipam`" in the comment. Missing space before the open paren

Reply:
Fixed.